tests/components/holiday/test_calendar.py

```diff
@@ -0,0 +1,36 @@
+"""Tests for calendar platform of Holiday integration."""
+
+from datetime import datetime
+from unittest.mock import patch
+
+from homeassistant.components.holiday import calendar
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_async_get_events(hass: HomeAssistant):
+    """Test events in a specific time frame."""
+    config_entry = MockConfigEntry(
+        data={
+            "country": "US",
+            "province": "AK",
+        }
+    )
+    entity = calendar.HolidayCalendarEntity(
+        "United States, AK", "US", "AK", config_entry.entry_id
+    )
+
+    start_date = datetime(2023, 1, 1)
+    end_date = datetime(2023, 1, 2)
+
+    with patch(
```

Comment:
We could/should move time instead like we do in `workday`

Reply:
Done.